src/modules/SpaceCalculation.rb

```diff
@@ -825,27 +806,25 @@ def GetPartitionWarning
 
       Builtins.y2debug("Total used space (kB): %1", used)
 
-      if Ops.greater_than(Builtins.size(message), 0)
-        # dont ask user to deselect packages for imap server, product
-        if ProductFeatures.GetFeature("software", "selection_type") == :auto
-          message = if Mode.update
-            Builtins.add(
-              message,
-              "\n" +
-                # popup message
-                _(
-                  "Deselect packages or delete data or temporary files\n" \
-                    "before updating the system.\n"
-                )
-            )
-          else
-            Builtins.add(
-              message,
-              "\n" +
-                # popup message
-                _("Deselect some packages.")
-            )
-          end
+      if Ops.greater_than(Builtins.size(message),
+        0) && (ProductFeatures.GetFeature("software", "selection_type") == :auto)
```

Comment:
Um, this leading `0)` looks strange, should be on the previous line...

Reply:
agreed